bokehjs/src/lib/models/plots/plot_canvas.ts

```diff
@@ -574,6 +575,27 @@ export class PlotView extends LayoutDOMView implements Renderable {
 
     this.connect(this.model.change, () => this.request_paint("everything"))
     this.connect(this.model.reset, () => this.reset())
+
+    const {toolbar_location} = this.model.properties
```

Comment:
why is `.properties` needed here (and elsewhere)?

Reply:
Because this needs access to the actual property (e.g. its `change` signal) and not the value. As a refresher, `this.model.toolbar_location` is a shorthand for `this.model.properties.toolbar_location.get_value()`.

Reply:
I'm aware :) needing the property object itself was the part I missed